src/_routing/_components/AuthenticationNavigator.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from 'react';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+
+import Home from '../../home/Home';
+
+export type TAuthenticationParams = {
```

Comment:
niet consistent met hoe Helena het had gedaan

Reply:
Ik denk dat dit gewoon een foute merge is ofzo, want dit heb ik al aangepast in `develop` 🤔